virt/virt-4-13-release-notes.adoc

```diff
@@ -198,11 +198,19 @@ link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_mana
 
 * The virtual machine snapshot restore operation no longer hangs indefinitely due to some persistent volume claim (PVC) annotations created by the Containerized Data Importer (CDI). (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070366[*BZ#2070366*])
 
-
-
 [id="virt-4-13-known-issues"]
 == Known issues
 
+// CNV-29988 Reenable 'DisableMDEVConfiguration' feature gate
+* If you enabled the `DisableMDEVConfiguration` feature gate in {VirtProductName} 4.12.4, you must reenable the feature gate after upgrading to versions 4.13.0 or 4.13.1 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2184439[*BZ#2184439*]):
```

Comment:
Can we please update it to the below, 

`If you enabled the `DisableMDEVConfiguration` feature gate in {VirtProductName} 4.12.4 via HCO CR, you must reenable the feature gate after upgrading to versions 4.13.0 or 4.13.1 using the below JSON patch annotation.`

Reply:
I cannot use your original wording but I can preserve the meaning.